.github/workflows/integration-tests-api.yml

```diff
@@ -3,7 +3,7 @@ on: pull_request
 
 jobs:
   runTests:
-    name: Run API integration tests
+    name: API test
```

Comment:
![image](https://github.com/matter-labs/block-explorer/assets/51121511/90dba27d-13a4-47c9-a09d-33c9288edc91)
to have a full task name I would suggest to use a shorter name here